tests/drivers/dma/loop_transfer/boards/longan_nano.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2022 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&dma0 {
```

Comment:
Add GD32F450I-EVAL + GD32F403Z-EVAL + GD32F350R-EVAL + GD32E103V-EVAL + GD32VF103V-EVAL boards in the tests loop. The logan nano can be removed because we should try use always manufacturer boards here.

Reply:
Add boards that are in the following list.
- gd32e103v_eval
- gd32f350r_eval
- gd32f403z_eval
- gd32f450i_eval
- gd32f450z_eval
- gd32f470i_eval
- gd32vf103v_eval

But I don't have these boards. I only verified it is compilable.